docker/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+daemon="${1:-nginx}"
+
+if [[ "$daemon" = "nginx" ]]; then
+  nginx -s 'deamon off;'
+elif [[ "$daemon" = "ipfs" ]]; then
+  ipfs config --json Addresses.Gateway '"/ip4/127.0.0.1/tcp/80"'
```

Comment:
**Question (blocking)**: do we need to run ‘IPFS add’ here or is it already added/pinned?